Assets/MRTK/Providers/LeapMotion/Editor/LeapMotionConfigurationChecker.cs

```diff
@@ -403,21 +415,88 @@ static void UpdateCSC()
             Debug.Log($"Saving {cscFilePath}");
         }
 
-#if UNITY_2018       
+
         /// <summary>
-        /// Force Leap Motion integration after the Leap Motion Core Assets import.  In Unity 2018.4, the configuration checker sometimes does not update after the 
-        /// Leap Motion Core Assets import, this case only occurs if the MRTK source is from the unity packages. If the integration of leap and MRTK has not occurred, users can 
-        /// select the Configure Leap Motion menu option to force integration. 
+        /// Connect MRTK and the Leap Motion Unity Modules if the Leap Motion Unity Modules are in the project. If they are not in the project, display a pop up window.
         /// </summary>
-        [MenuItem("Mixed Reality Toolkit/Utilities/Leap Motion/Configure Leap Motion")]
-        static void ForceLeapMotionConfiguration()
+        [MenuItem("Mixed Reality Toolkit/Utilities/Leap Motion/Connect Leap Motion Unity Modules")]
+        public static void ConnectLeapMotionToMRTK()
```

Comment:
Do the menu items need to be made public for something else to call them? Or should they be left private so they're only call-able via the menu item?

Reply:
I made them public so that we can use them for a pipeline if needed

Reply:
Oh good point!